settings.py

```diff
@@ -285,6 +285,7 @@
     'corehq.apps.analytics',
     'corehq.apps.callcenter',
     'corehq.apps.change_feed',
+    'corehq.apps.cowin',
```

Comment:
Is this only used by a handful of domains? Did we consider putting this in `custom` instead of `corehq`, and making it available just to those domains, instead of creating a new feature flag?

Reply:
As of now it would be available for only one domain, which is the project that has the real Vaccination solution app. It probably could be added to some test domains.

The feature flag is so that the repeaters are only available for specific domain.
I am not sure how this would be added under `custom` to make it available for specific domains. But if that is the right way to go about this, happy to move it there.

Reply:
+1 this should have been under `custom`

Reply:
okay, not sure entirely how this is going to look like but started with a simple move first

https://github.com/dimagi/commcare-hq/pull/29990